src/calculator/ivs.ts

```diff
@@ -7,24 +7,24 @@ import { getCpm, getCp, getCpFromLevel } from './stats'
 export const getDefaultIvs = (
   dexEntry: PokedexEntry, cpLimit: number
 ): LevelStats => {
-  let stats: LevelStats = { level: 40, atk: 15, def: 15, hp: 15 }
+  let stats: LevelStats = { level: 40, atk: 15, def: 15, hp: 15 };
```

Comment:
Can we not use semicolons please? Changing the source to have them makes a lot of noise that makes it hard to see what has actually changed, unless we change the whole code base, and my strong preference is not to use them.